problems/double_pendulum/double_pendulum_symbolic.jl

```diff
@@ -0,0 +1,129 @@
+using RigidBodyDynamics, MeshCat, MeshCatMechanisms, SymPy #Symbolics
+using LinearAlgebra, StaticArrays, Random
+
+using Dionysos
+const DI = Dionysos
+const UT = DI.Utils
+const DO = DI.Domain
+const PB = DI.Problem
+const ST = DI.System
+const CO = DI.Control
+const SY = DI.Symbolic
+
+Random.seed!(123)
+
+include("double_pendulum_urdf.jl")
+
+# les moments d'inerties ne semblent pas etre les memes ?  q2(eux) = q2(moi)-q1
```

Comment:
It should be written in english, or be removed

Reply:
I removed